### ISSUE TITLE
"File Issue" share option

### ISSUE THREAD

**Issue by marcprux** [vega/editor#752]
Whenever I file a bug against vega or vega-lite, I include a spec and a screenshot from vega-editor to provide a reproducible test. This workflow is a process of manual copying/pasting and creating the screenshot and uploading it to a new issue report.

A "File Issue" link in the "Share" menu could streamline this:

• Create a new issue for vega or vega-lite
• Inline the formatted JSON spec in the body
• Include the screenshot as an attachment
• Include any additional vega-editor configuration options (such as which renderer is being used)
• Include the current version of vega, vega-lite, etc.
• Include information on the browser & OS that is being used

**Comment by domoritz** [comment]
Yep, that would be cool. As a first step, you can get a markdown link.

<img width="717" alt="Screen Shot 2020-08-03 at 10 06 28" src="https://user-images.githubusercontent.com/589034/89208337-02031a80-d571-11ea-9241-243d6ff72389.png">

The editor already has a little link in the help dialog that sets the version. 

<img width="804" alt="Screen Shot 2020-08-03 at 10 07 01" src="https://user-images.githubusercontent.com/589034/89208416-2a8b1480-d571-11ea-914b-5385f37285e8.png">

We could base your suggestion on what we did there.

**Comment by marcprux** [comment]
One additional nice feature would be to include the current log messages/warnings/errors in the issue template.
